pkg/util/paramtable/component_param.go

```diff
@@ -4563,6 +4557,12 @@ type dataNodeConfig struct {
 	BloomFilterApplyParallelFactor ParamItem `refreshable:"true"`
 
 	DeltalogFormat ParamItem `refreshable:"false"`
+
+	// index services config
+	BuildParallel          ParamItem `refreshable:"false"`
+	EnableDisk             ParamItem `refreshable:"false"`
+	DiskCapacityLimit      ParamItem `refreshable:"true"`
```

Comment:
is this configure item still in use?

Reply:
We will discuss and decide in the next PR whether to remove it.